### PR TITLE
Create preloader

### PR DESCRIPTION
**Pull request by DmitriyKrivovyaz**

### CODE REVIEW

File: client/src/components/Preloader/Preloader.jsx [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  );
+}
+
+Preloader.defaultProps = {
+    centralObj: moon,
+    rotatingObj: sun
+}
+
+Preloader.propTypes = {
+    centralObj: string,
```
Comment (anton-vlasik) [inline client/src/components/Preloader/Preloader.jsx]: For some reason obj is a string... but why?

File: client/src/components/Preloader/Preloader.jsx [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+  return (
+    <div className={classes.container}>
+      <img src={centralObj} className={classes.centralObj} alt='cental obj'/>
+      <img src={rotatingObj} className={classes.rotatingObj} alt ='rotating obj'/>
+    </div>
+  );
+}
+
+Preloader.defaultProps = {
+    centralObj: moon,
```
Comment (anton-vlasik) [inline client/src/components/Preloader/Preloader.jsx]: it's not necessarily a good practice to use position related names for your objects

File: client/src/redux/actions/appInitializationAction.js [hunk 3]
```diff
@@ -10,8 +10,8 @@ export const initializeApp = async dispatch => {
   const location = `${currentLocation.coords.latitude},${currentLocation.coords.longitude}`;
 
   await getAndSetAccessToken();
-  dispatch(getLocationAction(currentLocation));
-  dispatch(getWeatherAction(location));
-  dispatch(getHourlyWeatherAction(location));
-  dispatch(getDailyWeatherAction(location));
+  await dispatch(getLocationAction(currentLocation));
```
Comment (anton-vlasik) [inline client/src/redux/actions/appInitializationAction.js]: 1) I don't think this change has something to do with this PR's topic
2) I don't see that any of subsequent requests is using data from previous one. Then the question is why not Promise.all ?

File: client/src/components/Preloader/preloader.module.css [hunk 4]
```diff
@@ -0,0 +1,30 @@
+.container {
+  width: 100%;
+  height: 100vh;
+  background-color: #2f1c5a;
```
Comment (ximet) [inline client/src/components/Preloader/preloader.module.css]: emmmm again? Where you color pallete?

Comment (DmitriyKrivovyaz) [reply client/src/components/Preloader/preloader.module.css]: moved to index.css

File: client/src/components/Preloader/Preloader.jsx [hunk 5]
```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import classes from './preloader.module.css';
+import sun from '../../../public/images/sun.png';
+import moon from '../../../public/images/moon.png';
+import PropTypes from 'prop-types';
+
+function Preloader({ staticImage, rotatingImage, theme }) {
+  return (
+    <div className={classes.container} data-theme={theme}>
+      <img src={staticImage} className={classes.staticImage} alt="cental obj" />
```
Comment (anton-vlasik) [inline client/src/components/Preloader/Preloader.jsx]: 1) Still not correct to my mind. src={imageSrc}
2) So you want to make it flexible and pass some image sources from above... but then again you are using rotating, static prefixes which indicate that they have bound yourself to specific look. Not good. Either hardcode them here or make it really flexible

**Review by ximet — APPROVED** [review]

File: client/src/components/Preloader/Preloader.jsx [hunk 6]
```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import classes from './preloader.module.css';
+import sun from '../../../public/images/sun.png';
+import moon from '../../../public/images/moon.png';
+import PropTypes from 'prop-types';
+import sun from '../../../public/images/sun.png';
+import moon from '../../../public/images/moon.png'
```
Comment (anton-vlasik) [inline client/src/components/Preloader/Preloader.jsx]: double import

File: client/src/components/Preloader/Preloader.jsx [hunk 7]
```diff
@@ -0,0 +1,22 @@
...
+
+function Preloader({ theme }) {
+  return (
+    <div className={classes.container} data-theme={theme}>
+      <img src={moon} className={classes.staticImage} alt="cental obj" />
+      <img src={sun} className={classes.rotatingImage} alt="rotating obj" />
```
Comment (anton-vlasik) [inline client/src/components/Preloader/Preloader.jsx]: animated would sound more generic

**Review by anton-vlasik — APPROVED** [review]

**Pull request merged**